java/arcs/core/entity/Handle.kt

```diff
@@ -12,16 +12,31 @@
 package arcs.core.entity
 
 import arcs.core.data.HandleMode
+import kotlin.coroutines.resume
+import kotlin.coroutines.suspendCoroutine
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Job
 
 /** Base interface for all handle classes. */
 interface Handle {
     val name: String
 
+    /**
+     * If you need to access or mutate the [Handle]'s data from outside of a particle or handle
+     * lifecycle callback, you should use this dispatcher to run your operations.
```

Comment:
s/should/must ?

Reply:
Done! also added a TODO to add a dispatcher to the particle interface.